cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/remote/RemoteOpsClient.java

```diff
@@ -73,6 +89,24 @@ private String resolve(String resource) {
                          addr.getHostName(), addr.getPort(), "/v1", resource);
   }
 
+
+  protected void executeRequest(String methodName, Object... arguments) {
+    doPost("execute/" + methodName, GSON.toJson(createArguments(arguments)));
+  }
+
+  private static List<MethodArgument> createArguments(Object... arguments) {
```

Comment:
doesn't need to be static, but I think @Nishith had suggested it in previous PR.
It makes sense to indicate that the method doesn't rely on nor affect state.